subiquity/server/controllers/drivers.py

```diff
@@ -69,7 +73,24 @@ def start(self):
         self.app.hub.subscribe(
             InstallerChannels.APT_CONFIGURED,
             self._wait_apt.set)
```

Comment:
Looking at this line... are we sure that APT_CONFIGURED is going to fire only once in the future? It's certainly true today, so maybe this is a worry for another day.

Reply:
I think the answer is yes as long as we fire this event after _successfully_ determining which mirror we will use.